### ISSUE TITLE
Update Pod version

### ISSUE THREAD

**Issue by josh-byster** [sudeepag/SAConfettiView#45]
It looks like the changes for Swift 4 compatibility have been merged in a PR, but right now running `pod install` will result in version 0.1.8 being installed, which doesn't have the Swift 4 compatibility.

**Comment by tazmancoder** [comment]
Yes, CocoaPods is still not updated. Also in the code they still need to replace the following line:
Replace 'kCAEmitterLayerLine' with 'CAEmitterLayerEmitterShape.line'

**Comment by ugurethemaydin** [comment]
Unfortunately, this library has not been updated for a long time and does not work after swift 4.2.
If you get an error and cannot use this library, you can visit it.
For the active repo and many new features,

https://github.com/ugurethemaydin/SwiftConfettiView
